src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -12,6 +12,43 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
     [CompilerTrait(CompilerFeature.ReadOnlyReferences)]
     public class RefEscapingTests : CompilingTestBase
     {
+        [Fact]
+        public void RefStructSemanticModel()
+        {
+            var tree = SyntaxFactory.ParseSyntaxTree(@"
+using System;
+struct S1 { }
+ref struct S2 { public S1 F1; }
+class C<T>
+{
+    void M()
+    {
+        Span<int> span = stackalloc[0];
+        var s1 = new S1();
+        var s2 = new S2();
+        var i0 = 0;
+        var t1 = default(T);
+    }
+}");
+            var comp = CreateCompilationWithSpan(tree);
+            var model = comp.GetSemanticModel(tree);
+            var root = tree.GetRoot();
+
+            Assert.True(getLocalType("span").IsRefLikeType);
```

Comment:
There are implementations of `IsRefLikeType` on many type symbols (anonymous types, arrays, wrapped types like tuples, pe named types, etc). They should be covered too.

Reply:
I've presumed that there are numerous tests using that API already. The interface implementation is implicit and uses the same API. It seems a little bit overboard to check the same thing for all ITypeSymbols as well.